packages/components/addon/components/hds/dropdown/list-item/checkbox.js

```diff
@@ -4,7 +4,7 @@
  */
 
 import Component from '@glimmer/component';
-import { getElementId } from '../../form/utils/getElementId';
+import { getElementId } from '../../../../helpers/hds-get-element-id';
```

Comment:
```suggestion
import { getElementId } from '@hashicorp/design-system-components/helpers/hds-get-element-id';
```
it's slightly more typing, but my experience is that this other syntax utilizing ember's resolver is a little more digestible/maintaible than trying to figure out how many times you have to traverse up the file tree to find this file

Reply:
I was looking for a quicker/easier to read resolver on this, but couldn't figure it out